ycms/cms/models/bed_assignment.py

```diff
@@ -86,17 +88,19 @@ def until_discharge(self):
         :return: the current bed assignment until discharge
         :rtype: str
         """
+
         if self.duration is None or self.discharge_date is None:
             return None
 
         until_discharge = int(
             ((self.discharge_date.date() - current_or_travelled_time().date()).days)
         )
 
+        in_until_discharge_days = _("in {} days").format(until_discharge)
+        in_until_discharge_day = _("in {} day").format(until_discharge)
+
         return (
-            f"{_('in')} {until_discharge} {_('days')}"
-            if until_discharge > 1
-            else f"{_('in')} {until_discharge} {_('day')}"
+            in_until_discharge_days if until_discharge > 1 else in_until_discharge_day
```

Comment:
```suggestion
            in_until_discharge_days if until_discharge == 1 else in_until_discharge_day
```
I think in both English and German it's "in 0 days" / "in 0 Tagen", right?